src/laserscan_multi_merger.cpp

```diff
@@ -206,6 +206,13 @@ void LaserscanMerger::pointcloud_to_laserscan(pcl::PCLPointCloud2 *merged_cloud)
 			continue;
 		}
 
+		double range_max_sq_ = output->range_max * output->range_max;
+		if (range_max_sq_ <= range_sq)
```

Comment:
I was wondering if it would be `<` instead of `<=` if it was to align with line 203.
It's trivial and I may have misunderstood something, so Approve.

203行目と揃えるなら、`<=`ではなく、`<`になるのかなと思いました。
些細ですし、何か勘違いしているかもなので、Approveします。

Reply:
@kokosabu 
min が < で max が <= になるのは、後続処理でScanの点群の処理対象になるのが、
range_min 以上 かつ range_max 未満の時なので、その範囲内に収める為です。
※ range_max が 30.0 の時に range が 30.0 の点があると処理対象外になる

https://github.com/ros-perception/laser_geometry/blob/kinetic-devel/src/laser_geometry.cpp#L422